TruePath/IPath.cs

```diff
@@ -48,3 +48,33 @@ public interface IPath<TPath> where TPath : IPath<TPath>
     /// <remarks>Note that currently this comparison is case-sensitive.</remarks>
     bool StartsWith(TPath other);
 }
+
+public interface IPathWithoutOperator<TPath>
```

Comment:
What is the point of this interface? Could you please give examples of its usage?